test/src/test/java/hudson/cli/DeleteNodeCommandTest.java

```diff
@@ -48,7 +48,7 @@
 
     @Before public void setUp() {
 
-        command = new CLICommandInvoker(j, new DeleteNodeCommand());
+        command = new CLICommandInvoker(j, "delete-node");
```

Comment:
It reflects my comment from the original PR. The effect is the same with the exception this resolved the command even when registered using `CLIMethod` (and therefore do not have a class). This is useful for refactoring `CLIMethod`s to `CLICommand`s as one can checkout implementation back and forth to run tests and ensure the refactoring have not changed the behaviour.